pkg/sanity/sanity.go

```diff
@@ -40,6 +40,8 @@ type CSISecrets struct {
 	ControllerUnpublishVolumeSecret map[string]string `yaml:"ControllerUnpublishVolumeSecret"`
 	NodeStageVolumeSecret           map[string]string `yaml:"NodeStageVolumeSecret"`
 	NodePublishVolumeSecret         map[string]string `yaml:"NodePublishVolumeSecret"`
+	CreateSnapshotSecret            map[string]string `yaml:"CreateSnapshotSecret"`
+	DeleteSnapshotSecret            map[string]string `yaml:"DeleteSnapshotSecret"`
```

Comment:
Along with these for snapshot secrets in the mock driver, you should also add snapshot secret authentication in driver/driver.go, like https://github.com/kubernetes-csi/csi-test/blob/master/driver/driver.go#L198-L204.
And then some snapshot secrets in https://github.com/kubernetes-csi/csi-test/blob/master/mock/mocksecret.yaml. Those would be used by the mock test run with secrets enabled,

Reply:
Thanks for your help, Updated.